lib/tasks/technical_metadata.rake

```diff
@@ -3,7 +3,7 @@
 namespace :techmd do
   desc 'Generate technical metadata (synchronously) from druid + filepaths'
   task :generate, %i[druid filepaths basepath force] => :environment do |_task, args|
-    filepath_map = FilepathSupport.filepath_map_for(filepaths: args[:filepaths].split(','), basepath: args[:basepath])
+    filepath_map = FilepathSupport.filepath_map_for(filepaths: args[:filepaths].split, basepath: args[:basepath])
```

Comment:
split list of filenames on space as shown in readme example and not comma (comma doesn't work when passing in command line args anyway unless you escape them)